src/components/apiproduct/APIProductForm.tsx

```
@@ -483,6 +484,16 @@ const APIProductForm: React.FC<APIProductFormProps> = ({
                 description={t('Keys are created without need to be approved.')}
               />
             </FormGroup>
+            {isEditMode && obj?.spec?.approvalMode === 'manual' && formData.approvalMode === 'automatic' && (
```

ESLint

[error] Replace `·obj?.spec?.approvalMode·===·'manual'·&&` with `⏎··············obj?.spec?.approvalMode·===·'manual'·&&⏎·············`

(prettier/prettier)